### PR TITLE
[KYUUBI #4770][1.7] Compilation fails with AdminResourceSuite

### LINKED ISSUE #4770: [Bug] Compilation fails with `AdminResourceSuite` on branch 1.7
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/kyuubi/issues?q=is%3Aissue) and found no similar issues.

### Describe the bug

https://github.com/apache/kyuubi/pull/4740#issuecomment-1521342224

### Affects Version(s)

1.7

### Kyuubi Server Log Output

_No response_

### Kyuubi Engine Log Output

_No response_

### Kyuubi Server Configurations

_No response_

### Kyuubi Engine Configurations

_No response_

### Additional context

_No response_

### Are you willing to submit PR?

- [ ] Yes. I would be willing to submit a PR with guidance from the Kyuubi community to fix.
- [X] No. I cannot submit a PR at this time.

### PR DESCRIPTION
**Pull request by zwangsheng**
### _Why are the changes needed?_

Close #4770

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request
- [x] [CI]

### CODE REVIEW

**Comment by zwangsheng** [comment]
cc @bowenliang123, as this block 1.7.1 release progress

File: kyuubi-server/src/test/scala/org/apache/kyuubi/server/api/v1/AdminResourceSuite.scala [hunk 1]
```diff
@@ -177,6 +178,12 @@ class AdminResourceSuite extends KyuubiFunSuite with RestFrontendTestHelper {
       assert(client.pathExists(engineSpace))
       assert(client.getChildren(engineSpace).size == 1)
 
+      val adminUser = Utils.currentUser
+      val encodeAuthorization = new String(
+        Base64.getEncoder.encode(
+          s"$adminUser:".getBytes()),
+        "UTF-8")
```
Comment (pan3793) [inline kyuubi-server/src/test/scala/org/apache/kyuubi/server/api/v1/AdminResourceSuite.scala:185]: merge L184 and L185 to L183

Comment (zwangsheng) [reply kyuubi-server/src/test/scala/org/apache/kyuubi/server/api/v1/AdminResourceSuite.scala]: done

**Comment by bowenliang123** [comment]
How about extracting `encodeAuthorization` as a public method like you did in the previous commit?
https://github.com/apache/kyuubi/commit/f6331a2a0fc0650ba0970d58f90a7a7c8e908095#diff-0d114337935f257a9589963c4938c9831ad99dd11531af8f7f1e1dce0da01aa7R53

**Comment by zwangsheng** [comment]
> How about extracting encodeAuthorization as a public method like you did in the previous commit, rather than repeating them?
https://github.com/apache/kyuubi/commit/f6331a2a0fc0650ba0970d58f90a7a7c8e908095#diff-0d114337935f257a9589963c4938c9831ad99dd11531af8f7f1e1dce0da01aa7R53

Fine to this

File: kyuubi-server/src/test/scala/org/apache/kyuubi/server/api/v1/AdminResourceSuite.scala [hunk 2]
```diff
@@ -32,12 +32,16 @@ import org.apache.kyuubi.ha.HighAvailabilityConf
...
 import org.apache.kyuubi.plugin.PluginLoader
+import org.apache.kyuubi.server.KyuubiRestFrontendService
 import org.apache.kyuubi.server.http.authentication.AuthenticationHandler.AUTHORIZATION_HEADER
 
 class AdminResourceSuite extends KyuubiFunSuite with RestFrontendTestHelper {
 
   private val engineMgr = new KyuubiApplicationManager()
 
+  val encodeAuthorization =
+    new String(Base64.getEncoder.encode(s"${Utils.currentUser}:".getBytes()), "UTF-8")
```
[the remark this answers is not in the document]
Comment (zwangsheng) [reply kyuubi-server/src/test/scala/org/apache/kyuubi/server/api/v1/AdminResourceSuite.scala]: done

**Comment by bowenliang123** [comment]
LGTM. Have you tested compilation with branch-1.7 ?

**Comment by zwangsheng** [comment]
> LGTM. Have you tested compilation with branch-1.7 , as we don't have ci build for that ?

Nope, i will run locally

**Comment by bowenliang123** [comment]
Thanks for the fix. Merged to 1.7.1 .
